azure-cosmosdb-table/setup.py

```diff
@@ -68,6 +68,7 @@
     packages=find_packages(),
     install_requires=[
                          'azure-common>=1.1.5',
+                         'azure-storage-common',
```

Comment:
Please consider setting a version range for the dependency like [here](https://github.com/Azure/azure-storage-python/blob/master/azure-storage-blob/setup.py#L89).